packages/opentelemetry-scope-base/src/NoopScopeManager.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright 2019, OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as types from './types';
+
+export class NoopScopeManager implements types.ScopeManager {
+  active(): unknown {
+    return null;
+  }
+
+  with<T extends (...args: unknown[]) => ReturnType<T>>(
+    scope: unknown,
+    fn: T
+  ): ReturnType<T> {
+    return fn();
+  }
+
+  bind<T>(target: T, scope?: unknown): T {
+    return target;
+  }
+
+  enable(): this {
+    return this;
+  }
+
+  disable(): void {}
```

Comment:
This should be consistent with `enable`, so both should return `this` or `void` but not a mix.